bundle/manifests/windows-prometheus-k8s-rules_monitoring.coreos.com_v1_prometheusrule.yaml

```diff
@@ -39,3 +39,9 @@ spec:
     - expr: |
         windows_cpu_info
       record: node_cpu_info
+    - expr: |
+        sum(rate(windows_container_cpu_usage_seconds_total[5m]) * on(container_id) group_left(namespace, pod, container) kube_pod_container_info{container_id!=""}) by (pod,namespace)
```

Comment:
I may have to test this out but AFAIK pod label is not available through windows_exporter, an option here to remove ` * on(container_id) ` is to do a label mapping in:
[bundle/manifests/windows-exporter_monitoring.coreos.com_v1_servicemonitor.yam]
````
relabelings:
        - action: replace
          regex: (.*)
          replacement:
          sourceLabels:
            - __meta_kubernetes_endpoint_address_target_name
          targetLabel: instance
        - action: labelmap
          source_labels: [container_id]
          replacement: pod`
```

Reply:
correct Windows exporter does not add the pod label
I am adding it via [this method](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.container.md#useful-queries)